docker/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.10
+
+WORKDIR /webui
+
+RUN apt-get update && \
+    apt-get install ffmpeg libsm6 libxext6 dos2unix google-perftools -y
```

Comment:
It is also suggested to add `apt-get clean` or `rm -rf /var/cache/apt/archives /var/lib/apt/lists/*` here to remove any cached files that will just take up space for no reason.